src/main/java/command/ToDoCommand.java

```diff
@@ -0,0 +1,53 @@
+package command;
+
+import exception.ToDoException;
+import storage.Storage;
+import task.Task;
+import task.TaskList;
+import task.ToDo;
+import ui.Ui;
+
+/**
+ * Command for creating a ToDo Task.
+ */
+public class ToDoCommand extends Command {

```

Comment:
maybe you can rename this class as 'TodoCommand' to follow the naming convention?